api/db/services/artifact_commit_service.py

```
@@ -0,0 +1,204 @@
+            return None
+        if not row:
+            return None
+        # Attach nickname for symmetry with list_for_page.
+        nickname = ""
+        if row.get("user_id"):
+            try:
+                u = User.get_or_none(User.id == row["user_id"])
+                if u is not None:
+                    nickname = u.nickname or ""
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)